layouts/base.pug

```diff
@@ -205,7 +226,93 @@ html(lang='en')
+                        
+                        if (typeof theBrowser != "undefined") {
+                            if (theBrowser.os.group == "Windows") {
+                                // If the user is on Windows, get the tab index for the first "Windows" tab
+                                osSearchString = 'Windows';
+                            } else if (theBrowser.os.group == "Mac OS") {
+                                // If the user is on Mac, get the tab index for the first "Mac" tab
+                                osSearchString = 'Mac';    
+                            } else if (theBrowser.os.group == "Linux") {
+                                // If the user is on Linux, get the tab index for the first "Linux" tab
+                                osSearchString = 'Linux';
+                            }
+                            var osTab = $(this).find("a:contains('" + osSearchString + "')").first();
+                            var tabIndex = osTab.parent().index();
+                            $(this).tabs("option", "active", tabIndex);
+                        }
+                    });
+                });
+                
+                // Initialize the table of contents if it exists
+                $(function() {
+                    var toc = $("#toc").tocify({ context: ".c-article", 
+                                                 selectors: "h2,h3",
+                                                 showAndHide: false,
+                                                 smoothScroll: true,
+                                                 scrollTo: 0,
+                                                 showAndHideOnScroll: false,
+                                                 highlightOnScroll: true,
+                                                 highlightOffset: 10,
+                                                 extendPage: true,
+                                                 history: true,                               
+                                                 hashGenerator: "pretty",
+                                                 highlightDefault: false,
+                                                 ignoreSelector: "h4,h5,h6",
+                                                 scrollHistory: false
+                                                 });
+                    
+
+                    // If there is a TOC and a hash in the URL, click that TOC link automatically
+                    if (toc.length > 0 && window.location.hash !== '') {
+                        var hash = window.location.hash.substr(1);
+                        $('.tocify-item[data-unique="' + hash + '"]').click();
+                    }
+                });
+
+                // Bind a click handler to highlight the currently selected component in the navbar
+                $("a.sdk-sidebar__list-link[href*='components']").click(function() {
```

Comment:
I don't know what to do here. Is the issue with line 294? Should I add a new class to all links in the sidebar called `js-handle-active`?

Reply:
Yup, so instead of...

```
$("a.sdk-sidebar__list-link[href*='components']")
```

...do

```
$('.js-handle-active')
```

This would require updating all links yes, but you can do that in one place where `sdk-sidebar__list-link` is set in the markup.